android-client-sdk/src/main/java/com/devcycle/sdk/android/eventsource/EventHandler.kt

```diff
@@ -0,0 +1,71 @@
+package com.devcycle.sdk.android.eventsource
+
+import timber.log.Timber
+import com.devcycle.sdk.android.eventsource.EventHandler
+
+class Handler: EventHandler {
+    fun init() {}
+
+    /**
+     * EventSource calls this method when the stream connection has been opened.
+     * @throws Exception throwing an exception here will cause it to be logged and also sent to [.onError]
+     */
+    override fun onOpen() {
+        println("OPEN")
```

Comment:
same with these ones - we probably only want timber right?

Reply:
yeah, gotta clean these up